contents/tutorials/react-ab-testing.md

```diff
@@ -0,0 +1,165 @@
+
+function App() {
+	const posthog = usePostHog()
+
+const handleClick = () => {
+    posthog.capture('button clicked');
+  };
+
+  return (
+    <div className="App">
+      <h1>We are testing this button:</h1>
+      <button onClick={handleClick}>Click me!</button>
+    </div>
+  );
+}
+
+export default App;
+```
+
+Once done, go back to your app, click the button, and you should see a `button clicked` event captured into your PostHog instance.
+
+![Button clicked event](../images/tutorials/react-ab-testing/event.png)
+
+## Creating an experiment
+
+With PostHog set up, your React app is ready for the A/B test. To create one, go to the [experiments tab](https://app.posthog.com/experiments) in PostHog and click "New experiment." Add a name, feature flag key (like `home-button-test`), set your goal type to a trend of the `button clicked` event, and press "Save as draft." 
+
+![A/B test set up](../images/tutorials/react-ab-testing/ab-test.png)
+
+You can customize it further with a description, secondary metrics, or more variants (for [A/B/n testing](/tutorials/abn-testing)). 
+
+## Implementing the experiment
+
+With our experiment created, it’s time to add it to our React app. To do this, check the `home-button-test` flag using the `useFeatureFlagVariantKey` hook and show our changed content if it returns the value "test."
+
+```js
+// src/App.js
+import './App.css';
+import { useFeatureFlagEnabled, PostHogFeature, usePostHog, useFeatureFlagVariantKey } from 'posthog-js/react';
+
+function App() {
+  
+  const posthog = usePostHog();
+
+  posthog.featureFlags.override('home-button-test', 'test')
+  const flagValue = useFeatureFlagVariantKey('home-button-test')
+
+  const handleClick = () => {
+    posthog.capture('button clicked');
+  };
+
+  return (
+    <div className="App">
+      <h1>We are testing this button:</h1>
+      {flagValue === 'test' ? (
+        <button onClick={handleClick}>Sign up for free!</button>
+      ) : (
+        <button onClick={handleClick}>Click me!</button>
+      )}
+    </div>
+  );
+}
+
+export default App;
+```
+
+> Alternatively, you can use the **React feature component** as shown in the [How to set up React feature flags](/tutorials/react-feature-flags#using-the-posthog-feature-component) tutorial.
+
+To test that it works, override the feature flag with the test value.
+
+```js
+//...
+posthog.featureFlags.override({'home-button-test': 'test'})
+const flagValue = useFeatureFlagVariantKey('home-button-test')
+//...
+```
+
+After confirming it works, remove the `featureFlags.override` call, go back to your experiment in PostHog, and click "Launch." The A/B test is then running to get results on which version is best.
+
+## Further reading
+
+- [How YC's biggest startups run A/B tests (with examples)](/blog/ab-testing-examples)
+- [A software engineer's guide to A/B testing](/blog/ab-testing-guide-for-engineers)
+- [How to set up A/B/n testing](/tutorials/abn-testing)
```

Comment:
I think it could be better to link to tutorials here, especially react specific/ experiment specific

Reply:
updated